OpenRA.Game/Settings.cs

```diff
@@ -336,9 +336,16 @@ public class ChatSettings
 		public bool ConnectAutomatically = false;
 	}
 
+	public class LobbySettings
+	{
+		public string Faction = "Random";
```

Comment:
Color is already per mod. Player section (Name, Color, Server) is now also in mod settings, it overwrites main Player settings so it could be removed from settings. I kept it now so current user won't lose their settings. They are used first time and then saved also in mod settings to be used next time.